pkg/storage/redis/hmap.go

```diff
@@ -33,7 +33,7 @@ func (db *Redis) GetHMValue(entity, key string, value interface{}) error {
 func (db *Redis) AddHM(entity, key string, value interface{}) error {
 	j, err := json.Marshal(value)
 	if err != nil {
-		return errors.E(err, errors.Params{"value": value})
+		return errors.E(err, errors.Params{"key": key})
 	}
```

Comment:
Can we log the value here so that at least internally we know what failed?

Reply:
the package storage is not in charge to log something, this is only a wrapper to Redis database, he doesn't know about the log package. The callers of this package need to handle the error and log them

Reply:
Ok just want to make sure that the consumer of this is logging `value` so we can determine what is passing invalid JSON.